e2e/various-suite/return-to-previous.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { e2e } from '../utils';
+
+describe('ReturnToPrevious button', () => {
+  before(() => {
```

Comment:
can we provision a dashboard/alert rule instead? 🤔 this setup + teardown is more than the actual test and will take the majority of the time 😬